instrumentation.md

```diff
@@ -0,0 +1,49 @@
+## Instrumentation
+
+Bjoern can export connection and request metrics to statsd. If you want support for instrumentation you must compile
+bjoern manually with
+
+```bash
+BJOERN_WANT_STATSD=yes python setup.py install
+# or
+WANT_STATSD=yes make all
+```
+
+you can also enable Dogstatsd tags support by compiling with
+
+```bash
+BJOERN_WANT_STATSD=yes BJOERN_WANT_STATSD_TAGS=yes python setup.py install
+# or
+WANT_STATSD=yes WANT_STATSD_TAGS=yes make all
+```
+
+After building with statsd support you can enable statsd metrics on server boot by setting `enable_statsd=True`, e.g.
+
+```python
+bjoern.listen(wsgi_application, host, port)
+bjoern.run(statsd={'enable': True, 'host': '...', port: ..., ns: 'bjoern'})
+
+# With tags
+bjoern.run(statsd={'enable': True, 'host': '...', port: ..., ns: 'bjoern', 'tags': 'app:my-app-name,zone:central-europe'})
```

Comment:
I think it is worth clarifying that `tags` is expected to be a comma separated string and not a list of tags. Applications generally expect a list of string so people might struggle here.

Reply:
From what I've seen on the internet the tags format isn't standardised. I've seen versions like `"foo,bar,spam"` and versions like `"foo:x,bar:y,spam:z"`.

Agree with adding a comment!